api_schemas/user_schemas.py

```diff
@@ -29,6 +29,12 @@ class UserRead(fastapi_users_schemas.BaseUser[int], BaseSchema):
     want_notifications: bool
 
 
+class UserInGroupRead(fastapi_users_schemas.BaseUser[int], BaseSchema):

```

Comment:
Also not sure about `fastapi_users_schemas.BaseUser[int]` here, is this something we always want?